README.md

```diff
@@ -98,7 +98,7 @@ function initFingerprintJS() {
     .then((fp) =>
       fp.get({
         tag: {
-          deviceId: window.fingerprintjs.vendorId, // use vendor ID as device ID
+          deviceId: window.fingerprintjs, // use vendor ID as device ID
```

Comment:
`window.fingerprintjs.vendorId`

Reply:
fixed in 0464909

Reply:
No, it's not working this way.
Take a look: https://eager-hermann-4ea017.netlify.app/

Reply:
I think your example is quite different. I change the one above

Reply:
Since we are suggesting to declare the `window.fingerprintjs` as an object with the `vendorId` property

Reply:
sounds good, ty